src/command/format_stdin.rs

```diff
@@ -21,22 +22,10 @@ pub fn format_stdin_cmd(
         }
     };
 
-    // Search for the treefmt.toml from there.
-    let config_file = match config::lookup(work_dir) {
-        Some(path) => path,
-        None => {
-            return Err(anyhow!(
-                "{} could not be found in {} and up. Use the --init option to create one.",
-                config::FILENAME,
-                work_dir.display()
-            ));
-        }
-    };
-
     // Default the tree root to the folder that contains the config file
     let tree_root = tree_root.clone().unwrap_or_else(|| {
         // unwrap: since the config_file is a file, there MUST be a parent folder.
-        config_file.clone().parent().unwrap().to_path_buf()
+        config_file.parent().unwrap().to_path_buf()
```

Comment:
Now same as

https://github.com/numtide/treefmt/blob/d87045a6f3f6b2cbf85e1ae1eb10d548dd51829a/src/command/format.rs#L30

resolves

https://github.com/numtide/treefmt/actions/runs/4649699136/jobs/8228370543

```
error: using `clone` on a double-reference; this will copy the reference of type `&std::path::Path` instead of cloning the inner type
  --> src/command/format_stdin.rs:28:9
   |
28 |         config_file.clone().parent().unwrap().to_path_buf()
```